tests/components/AttachResult/AttachResult.spec.ts

```diff
@@ -1,11 +1,19 @@
-import { AttachResult, IAttachResultOptions } from '../../src/components/AttachResult/AttachResult';
+import { AttachResult, IAttachResultOptions } from '../../../src/components/AttachResult/AttachResult';
 import { Mock, Fake } from 'coveo-search-ui-tests';
-import { IQueryResult } from 'coveo-search-ui';
+import { IQueryResult, Logger } from 'coveo-search-ui';
 
 describe('AttachResult', () => {
     let attachResult: Mock.IBasicComponentSetup<AttachResult>;
     let fakeResult = Fake.createFakeResult();
 
+    beforeAll(() => {
```

Comment:
Order like this:

beforeAll
beforeEach
afterEach
afterAll
Because they're executed in this order.